testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -1959,6 +1960,24 @@ private static void checkConcatFunc(SqlOperatorFixture f) {
     f.checkFails("^concat()^", INVALID_ARGUMENTS_NUMBER, false);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-5771">[CALCITE-5771]
+   * Apply two different NULL semantics for CONCAT function(enabled in MySQL、
```

Comment:
I think it would be better to use here the "regular comma":  `,`  instead of `、`
The same applies for the similar messages in functions.iq, final commit message, PR title and Jira title.

Reply:
Done.